### PR TITLE
feat(ctr): Bundling, optimise assets, page weight/memory improvements

### PR DESCRIPTION
**Pull request by ashquarky**
Resolves #XXX

### Changes:

- Bundle ctr code. This allows for tree shaking etc. Including Pjax with the bundler also means one less http request during page load.
- Only deliver debug.js to non-console UAs. It had a check in it to do nothing on console, so why download the 25KiB?
- Re-created various images and theme assets to compress better. Stylesheet goes from 82KiB to 37KiB. Also included the raw assets in the repo, in case we swap to a better system of inlining.
- Calls cave.requestGc after PJAX page loads. This is supposed to help alleviate RAM issues on O3DS a bit, but more probably needs doing.

- [x] I have read and agreed to the [Code of Conduct](https://github.com/PretendoNetwork/Pretendo/blob/master/.github/CODE_OF_CONDUCT.md).
- [x] I have read and complied with the [contributing guidelines](https://github.com/PretendoNetwork/Pretendo/blob/master/.github/CONTRIBUTING.md).
- [ ] What I'm implementing was an [approved issue](../issues?q=is%3Aopen+is%3Aissue+label%3Aapproved).
- [x] I have tested all of my changes.

### CODE REVIEW

File: apps/juxtaposition-ui/src/services/juxt-web/views/ctr/root.tsx [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
 		<>
 			<link rel="stylesheet" type="text/css" href="/css/juxt.css" />
-			<script src="/js/debug.js"></script>
-			<script src="/js/pjax.js"></script>
-			<script src="/js/juxt.js"></script>
+			{/* Debug allows non-console browsers to have some amount of the cave API. */}
+			{props.ctx.uaIsConsole ?? true ? <></> : <script src="/js/debug.global.js"></script>}
+			<script src="/js/juxt.global.js"></script>
 		</>
 	);
```
Comment (mrjvs) [inline apps/juxtaposition-ui/src/services/juxt-web/views/ctr/root.tsx:16]: I had trouble understanding the logic, This should be the same outcome?
```suggestion
const addDebug = !props.ctx.uaIsConsole; // Add to non-console browsers
return (
<>
<link rel="stylesheet" type="text/css" href="/css/juxt.css" />
			{/* Debug allows non-console browsers to have some amount of the cave API. */}
			{addDebug ? <script src="/js/debug.global.js"></script> : null}
			<script src="/js/juxt.global.js"></script>
</>
);
```

File: apps/juxtaposition-ui/src/webfiles/ctr/js/debug.js [hunk 2]
```diff
@@ -1,17 +1,3 @@
...
-		if (evt.message) { // Chrome sometimes provides this
-			alert('error: ' + evt.message + ' at linenumber: ' + evt.lineno + ' of file: ' + evt.filename);
-		} else {
-			if ((evt.srcElement || evt.target).toString().indexOf('HTMLImageElement') !== -1) {
-				return;
-			}
-			alert('error: ' + evt.type + ' from element: ' + (evt.srcElement || evt.target));
-		}
-	}
-}
```
Comment (mrjvs) [inline apps/juxtaposition-ui/src/webfiles/ctr/js/debug.js:14]: What did this do?

File: apps/juxtaposition-ui/src/services/juxt-web/views/ctr/root.tsx [hunk 3]
```diff
@@ -1,17 +1,23 @@
...
+function DefaultHead(props: DefaultHeadProps): ReactNode {
 	return (
 		<>
 			<link rel="stylesheet" type="text/css" href="/css/juxt.css" />
-			<script src="/js/debug.js"></script>
-			<script src="/js/pjax.js"></script>
-			<script src="/js/juxt.js"></script>
+			{/* Debug allows non-console browsers to have some amount of the cave API. */}
+			{props.ctx.uaIsConsole ?? true ? <></> : <script src="/js/debug.global.js"></script>}
+			<script src="/js/juxt.global.js"></script>
```
Comment (mrjvs) [inline apps/juxtaposition-ui/src/services/juxt-web/views/ctr/root.tsx:14]: Where did `.global` come from, I don't see it in the compiling code anywhere

**Review by mrjvs — CHANGES_REQUESTED** [review]

File: apps/juxtaposition-ui/src/webfiles/ctr/js/debug.js (hunk 2 above)
Comment (ashquarky) [reply apps/juxtaposition-ui/src/webfiles/ctr/js/debug.js:14]: On 3DS consoles (cave not undefined) it would show an alert dialog for JavaScript errors. I had never seen nor heard of anyone seeing these alerts, and the plan was to stop shipping this file to 3DSes anyway, so I yoinked it.
We could add it back into juxt.js if it serves a useful debugging format.

File: apps/juxtaposition-ui/src/services/juxt-web/views/ctr/root.tsx (hunk 1 above)
Comment (ashquarky) [reply apps/juxtaposition-ui/src/services/juxt-web/views/ctr/root.tsx:16]: That seems more sensible yeah
